### DISCUSSION TITLE
Where to download hyg_v36_1.csv

### DISCUSSION THREAD

**Discussion by shahgazer** [cosinekitty/astronomy#358] [Q&A] [answered]
Hi there,

I just came across this library and trying out the python examples. While trying to run the stars_near_moon.py, it couldn't fine the hyg_v36_1.csv file under the "generate" folder.

FileNotFoundError: [Errno 2] No such file or directory: '../../generate/hyg_v36_1.csv'

Where can I download that particular file?

Thank you
Shah

**Comment by shahgazer** [comment]
Hi guys.

Anyone can help? Really appreciate it.

Thanks

**Comment by cosinekitty** [comment]
Sorry, I meant to respond to this earlier but I forgot. Here is where you get that file:

https://raw.githubusercontent.com/astronexus/HYG-Database/3964fa862d1f08f05919a35306889fa4a0afa7d6/hyg/v3/hyg_v36_1.csv

Contributors to Astronomy Engine already have this as a side-effect of running the unit tests. See [this line](https://github.com/cosinekitty/astronomy/blob/5d726f29784c209f9dc7ce50735e5e1577ef2f75/generate/run#L73C10-L73C129) of the `run` script.

**Reply by shahgazer** [reply, marked as answer]
No worries. Yes I got it working. Thanks!
